_includes/table.html

```diff
@@ -0,0 +1,51 @@
+<table>
+  <thead>
+    <tr>
+      <th style="text-align: left">Property</th>
+      <th style="text-align: center">Type</th>
+      <th style="text-align: left">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% assign props = include.props | split: ","  %}
+    {% for prop in props %}
+      {% assign type = include.source[prop].type %}
+      {% assign description =  include.source[prop].description" %}
+      {% if type == nil %}
+        <!--special case for $ref-->
+        {% if include.source[prop]["$ref"] %}
+          {% assign array = include.source[prop]["$ref"] | split: "/" %}
+          {% assign last_index = array | size | minus: 1 %}
+          {% assign type = array[last_index] %}
+
+        {% elsif include.source[prop].anyOf %}
+          <!--special case for anyOf-->
+          {% capture type %}
+            {% assign para = include.source[prop].anyOf %}
+            {% include type.html types=para %}
+          {% endcapture %}
+        {% endif %}
+      {% elsif type == "array" %}
+         <!--special case for type is arrays of objects -->
+        {% assign type = include.source[prop].items.type | append: "[]" | capitalize %}
+      {% elsif type.first != nil %}
+        <!--special case for type is multiple objects-->
```

Comment:
Do you mean if "type in the schema is an array of multiple types"?

If so, maybe update the comment a bit?